### DISCUSSION TITLE
I have wireshark pcaps but I am confused about the process to get this on kibana?

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#10752] [Unsupported Versions] [answered]
Hi, I used wireshark and winSCP to put a wireshark pcap on my dashboard from the link provided by so-import-pcap.    After that I am not sure what the process is to get it to display on the SOC? I used "sudo so-import-pcap. I see the md5 hash in /nsm/import. 
 
Is there a link maybe that I could look at? I have found different links, but none talking about trouble shooting this ..... yet. 

Thanks for any help or advice

**Comment by iqworks** [comment]
I found this link that is very interesting and might help some one who investigates this post:
https://www.elastic.co/blog/analyzing-network-packets-with-wireshark-elasticsearch-and-kibana 

thx

**Reply by iqworks** [reply]
I tried this link https://www.google.com/search?q=wireshark+security+onion+so-import-pcap+kibana&ei=B-WuZJflFKfbkPIP0ayUuA0&ved=0ahUKEwjXhM3x2omAAxWnLUQIHVEWBdcQ4dUDCBE&uact=5&oq=wireshark+security+onion+so-import-pcap+kibana&gs_lcp=Cgxnd3Mtd2l6LXNlcnAQAzIICAAQiQUQogQyBQgAEKIEMgUIABCiBDIFCAAQogQyBQgAEKIEOgoIABBHENYEELADOgUIIRCgAToKCCEQoAEQwwQQCkoECEEYAFDnBFiWhAFgvY8BaANwAHgAgAGcAYgBxBOSAQUxNS4xMZgBAKABAcABAcgBCA&sclient=gws-wiz-serp#fpstate=ive&vld=cid:2cb1e03d,vid:t7E0DPVa5t0 

from Doug. But when I clicked on the link generated by so-import-pcap it showed me no data on the dashboard? I am looking for trouble shooting advice at this point. 

thanks for any advice or suggestions

**Comment by phil1090** [comment, marked as answer]
What happens when you change the date filters in Hunt or Kibana to include the dates/times in the pcap?

**Reply by iqworks** [reply]
Hi phil1090, thanks for getting back. I DIDNT know you change the dates from WITHIN kibana and hunt! 
I believe this my be why kibana and hunt cannot see these pcaps. 
I will check on how to change the dates in those two tools. 

thanks again !!

**Comment by iqworks** [comment]
Hi phil1090, i finally changed the date in kibana and I can see them!
